### PR TITLE
Dept 30 add home button to navigation

### PR DESCRIPTION
**Pull request by blamboy090**
Updated styling for log out 
Added home navigational element with styling
While on home page button is highlighted

### CODE REVIEW

File: components/dept-heading/dept-heading.module.scss [hunk 1]
```diff
@@ -2,6 +2,8 @@ $font-family_1: Syne, Helvetica, Arial, sans-serif;
 
 .headingContainer {
   text-align: center;
+
```
Comment (lsanchezcode) [inline components/dept-heading/dept-heading.module.scss:5]: Maybe get rid of the extra space?

Comment (blamboy090) [reply components/dept-heading/dept-heading.module.scss]: Updated!

File: components/logout/logout.module.scss [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
   font-family: Inter, Helvetica, Arial, sans-serif;
   font-size: 1em;
-  font-weight: 500;
-  padding-top: 0.75em;
+  font-weight: 400;
   border-radius: 0.5em;
   justify-content: right;
-  margin-left: auto;
-  margin-top: 1.3em;
+  margin-left: 25px;
```
Comment (nkhorvash) [inline components/logout/logout.module.scss:9]: shouldn't this be in em?

File: components/home-button/home-button.module.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
+.homeContainer {
+  height: 0.55em;
+  font-family: Inter, Helvetica, Arial, sans-serif;
+  font-size: 1em;
```
Comment (nkhorvash) [inline components/home-button/home-button.module.scss:4]: Figma phase 3 says 14px font size (slightly less then 1em). Just wondering if that's a change Kathleen approved.

**Review by nkhorvash — CHANGES_REQUESTED** [review]
I'm glad to get the home button on the site, it helps a lot! The appearance is nice and it's keyboard navigable! One weird thing, line 35 of search-result/index.tsx has a console.log of the search results. This is not in sprint-2, but it doesn't show up in your diff??? Not sure how that works (yay git), but I don't think we want that in there.

File: components/logout/logout.module.scss (hunk 2 above)
Comment (blamboy090) [reply components/logout/logout.module.scss:9]: updated

File: components/home-button/home-button.module.scss (hunk 3 above)
Comment (blamboy090) [reply components/home-button/home-button.module.scss:4]: updated

**Review by nkhorvash — APPROVED** [review]
Everything looks good. The console.log is gone now, maybe the last time you pulled sprint-2 it was removed, or there was something weird in my local. Either way, looks good.

**Pull request merged**